src/main/scala/scalaz/ziokeeper/crdt/ORSet.scala

```diff
@@ -0,0 +1,33 @@
+package scalaz.ziokeeper.crdt
+
+import scalaz.ziokeeper.VectorClock
+
+case class ORSet[A, B](values: Vector[ORSet.VersionedEntry[A, B]])
+
+object ORSet {
+  implicit class ORSetOps[A, B](val oRSet: ORSet[A, B]) extends AnyVal {
```

Comment:
Maybe it makes sense to have these as methods in `ORSet`.

Reply:
sure